enterprise/frontend/src/metabase-enterprise/tenants/components/TenantSpecificCollectionsItemList/TenantSpecificCollectionsItemList.tsx

```diff
@@ -0,0 +1,62 @@
+import { useMemo } from "react";
+
+import { ItemList } from "metabase/common/components/EntityPicker/components/ItemList";
+import type {
+  CollectionItemListProps,
+  CollectionPickerItem,
+} from "metabase/common/components/Pickers/CollectionPicker/types";
+import { useListTenantsQuery } from "metabase-enterprise/api";
+import type { Tenant } from "metabase-types/api";
+
+export const TenantSpecificCollectionsItemList = ({
+  onClick,
+  selectedItem,
+  isFolder,
+  isCurrentLevel,
+  shouldDisableItem,
+  shouldShowItem,
+}: CollectionItemListProps) => {
+  const {
+    data: tenantsData,
+    error,
+    isLoading,
+  } = useListTenantsQuery({ status: "active" });
+
+  const tenantCollections = useMemo(
+    () => getTenantCollections(tenantsData?.data),
+    [tenantsData?.data],
+  );
+
+  return (
+    <ItemList
+      items={tenantCollections}
+      error={error}
+      isLoading={isLoading}
+      onClick={onClick}
+      selectedItem={selectedItem}
+      isFolder={isFolder}
+      isCurrentLevel={isCurrentLevel}
+      shouldDisableItem={shouldDisableItem}
+      shouldShowItem={shouldShowItem}
+    />
+  );
+};
+
+const getTenantCollections = (
```

Comment:
can't we just query collections in the `tenant-specific` namespace from the api instead of constructing a fake data structure from the list of tenants?